src/storage-types/src/dyncfgs.rs

```diff
@@ -75,13 +75,14 @@ pub const KAFKA_POLL_MAX_WAIT: Config<Duration> = Config::new(
     available.",
 );
 
-pub const KAFKA_ENDPOINT_IDENTIFICATION_ALGORITHM: Config<&'static str> = Config::new(
-    "kafka_endpoint_identification_algorithm",
-    // Default to no hostname verification, which is the default in versions of `librdkafka <1.9.2`.
-    "none",
-    "The value we set for the 'ssl.endpoint.identification.algorithm' option in the Kafka \
+pub const KAFKA_DEFAULT_AWS_PRIVATELINK_ENDPOINT_IDENTIFICATION_ALGORITHM: Config<&'static str> =
+    Config::new(
+        "kafka_default_aws_privatelink_endpoint_identification_algorithm",
+        // Default to no hostname verification, which is the default in versions of `librdkafka <1.9.2`.
+        "none",
+        "The value we set for the 'ssl.endpoint.identification.algorithm' option in the Kafka \
```

Comment:
Probably worth specifying in the description that this only happens for Kafka connections using default AWS PL.